genesis/experiment/rootdomain/rootdomain.go

```diff
@@ -148,6 +150,17 @@ func (rd *RootDomain) DumpAllUsers() []byte {
 	return resJSON
 }
 
+func (rd *RootDomain) SetRoot(adminKey string) string {
```

Comment:
If `SetRoot` must return reference of RootUser, it should be called GetRoot and return `rd.Root.String()` if its already set.
If its only create RootUser if needed, it shouldn't return anything.